source/server-discovery-and-monitoring/tests/errors/generate-error-tests.py

```diff
@@ -129,6 +130,7 @@ def create_stale_generation_tests():
     # Stale network errors
     for network_error_type, when in itertools.product(
             ['network', 'timeout'], WHEN):
+        error_name = network_error_type
```

Comment:
This is a minor fix for the `stale-generation-beforeHandshakeCompletes-timeout.yml`-like tests. It only changes the descriptions in the tests and doesn't change any behavior.